os_builders/galaxy.yml

```diff
@@ -0,0 +1,59 @@
+### REQUIRED
+
+# galaxy file for converting into a collection for Azimuth
+
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: azimuth
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: security
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 1.0.0
+
+# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
+readme: README.md
+
+# A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+# @nicks:irc/im.site#channel'
+authors:
+- your name <example@domain.com>
+
+
+### OPTIONAL but strongly recommended
+
+# A short summary description of the collection
+description: your collection description
```

Comment:
A collection of configuration options for STFC Cloud security compliance and modifications